routes/index.js

```diff
@@ -26,10 +26,13 @@ var importRoutes = keystone.importer(__dirname);
 keystone.pre('routes', middleware.initLocals);
 keystone.pre('render', middleware.flashMessages);
 
+// Import Route Controllers
+/*var routes = {
+	views: importRoutes('./views'),
+};*/
+
```

Comment:
why leave this here? remove it

Reply:
Will replace with routes in the next branch. Is having extra useless comments bad?